packages/grid/src/vaadin-grid-styles.js

```diff
@@ -140,6 +141,7 @@ registerStyles(
       align-items: center;
       padding: 0;
       white-space: nowrap;
+      pointer-events: all;
```

Comment:
Rather than duplicating this, define a common rule for cells:

```css
[part~='cell'] {
  pointer-events: initial;
}
```

Reply:
Addressed in the latest commit.